frontend/src/style/globals.css

```diff
@@ -5,7 +5,7 @@
 }
 
 * {
-  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Arial was set here because it's the font specified by the client - don't change the global.css settings without checking on this